keyboards/mechlovin/zed1800/zepsody/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+
+#define WEAR_LEVELING_BACKING_SIZE 4096
```

Comment:
actually, shouldn't this be set at the kb level?

Reply:
Thank you for your advice, will do.